src/MIDebugEngine/Natvis.Impl/Natvis.cs

```diff
@@ -624,10 +650,17 @@ private IVariableInformation[] ExpandVisualized(IVariableInformation variable)
                             if (arrayExpr.CountChildren != 0)
                             {
                                 uint offset = startIndex + requestedSize;
+                                bool isForward = item.Direction != ArrayDirectionType.Backward;
 
                                 for (uint index = 0; index < requestedSize; ++index)
                                 {
-                                    children.Add(new SimpleWrapper("[" + (startIndex + index).ToString(CultureInfo.InvariantCulture) + "]", _process.Engine, arrayExpr.Children[index]));
+                                    string displayName = (startIndex + index).ToString(CultureInfo.InvariantCulture);
+                                    if (rank > 0)
```

Comment:
Should this be `rank > 1`? Like isn't the index already correct for a rank 1 array?